ado-pipeline/build/docker-build.yaml

```
@@ -0,0 +1,30 @@
+parameters:
+  - name: dockerFilePath
+    type: string
+    default: "Dockerfile"
+steps:
+  - task: Docker@2
+    displayName: Login to ACR
+    inputs:
+      command: login
+      containerRegistry: AZD-FCP-ACR-SND2
+
+  - bash: |
+      if [ $isMainBranch == "true" ]; then
+        echo "Building Docker image for main branch"
+        tag=$(releaseVersion)
+      else
+        echo "Building Docker image for non-main branch"
+        tag="$(releaseVersion)-alpha.$(buildVer)"
+      fi
```

```suggestion
      fi
      echo "##vso[task.setvariable variable=tag]$tag"
```
I think you will need this to pass to the next task?